config/initializers/openai.rb

```diff
@@ -0,0 +1,6 @@
+if ENV.fetch("OPENAI_ACCESS_TOKEN", nil).present?
+  OpenAI.configure do |config|
+    config.access_token = ENV.fetch("OPENAI_ACCESS_TOKEN")
+    config.log_errors = true if Rails.env.development?
```

Comment:
You can do the following to simplify
```suggestion
    config.log_errors = Rails.env.development?
```